src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure implements FigureArea {
+    private final int height;
+    private final int width;
```

Comment:
let's better remove `final` from fields if they are not constants - everywhere in figure classes